In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

# Jaderná energie - kolik energie je v 1 kilogramu uranu?

V roce 2019 se na Slovensku spotřebovalo 30 103 GWh elektrické energie. Kolik kilogramů uranu $^{235}\text{U}$ jako jaderného paliva by bylo potřeba k vyprodujkování stejného pnožství energie? To zjistíme v tomto cvičení!

První si převedeme GWh na Jouli, se kterými běžně pracujeme. Jedna Wh odpovídá práci stroje s příkonem 1 watt po dobu 1 hodiny. To znamená, že $1 \text{ Wh} = 3600 \text{ J}$. Po použití tohoto převodního vztahu můžeme spotřebu elektrické energie vyjádřit v $J$.

\begin{equation}
    E = 30103 \text{ GWh} = 108370.8 \text{ TJ} = 108.3708 \cdot 10^{15} \text{J}
\end{equation}

V následujících výpočtech budeme přepokládat 100% účinnost, protože nám jde jen o odhad. 

Uran $^{235} \text{U}$ se při jaderném štěpení přemění na Barium $^{141}\text{Ba}$, Krypton $^{92}\text{Kr}$ a 2 volné neutrony. Kolik energie se při této přeměně uvolní? S tím nám pomůže snad nejslavněší rovnice fyziky

\begin{equation*}
    E = mc^2,
\end{equation*}

která dává do rovnosti hmotnost a energii. Podívejme se na hmotnosti jednotlivých částic a prvků vystupujících v této přeměně. Hmotnost vyjadřujeme v atomových hmotnostnbích jednotkách $m_u \approx 1.66 \cdot 10^{-27}$ kg.

\begin{equation*}
    ^{235}\text{U + } \text{n = } ^{141}\text{Ba + }  ^{92}\text{Kr} + \text{3n}
\end{equation*}

\begin{gather*}
    ^{235}\text{U = } 235.043928 \text{ } m_u, \\
    ^{141}\text{Ba = } 140.914411 \text{ } m_u, \\
    ^{92}\text{Kr = } 91.926156 \text{ } m_u, \\
    \text{n = } 1.008664 \text{ } m_u.
\end{gather*}

Když si hmotnosti na obou stranách rovnice sečteme, tak zjistíme, že hmotnosti nejsou stejné. To je dáno tím, že nukleony (protony a neutrony) "odevzdají" část svojí hmotnosti do vazební energie, která nuklenoy drží pohromadě. 

Zdroje

https://faktaoklimatu.cz/infografiky/elektrina-sr \
https://en.wikipedia.org/wiki/Nuclear_fission 

In [2]:
m_U_235 = 235.043928        # [m_u]
m_Ba_141 = 140.914411       # [m_u]
m_kr_92 = 91.926156         # [m_u]
m_n = 1.008664              # [m_u]
m_u = 1.66e-27              # [kg]

In [3]:
m_levo = m_U_235+m_n
m_pravo = m_Ba_141 + m_kr_92 + 3*m_n

print(f"levá strana rovnice má hmostnost {m_levo:,.2f} m_u a pravá strana rovnice má hmotnost {m_pravo:,.2f} m_u")

levá strana rovnice má hmostnost 236.05 m_u a pravá strana rovnice má hmotnost 235.87 m_u


Protože se hmotnost rovná energii podle rovnice $E = mc^2$ a energie se zachovává, tak se úbytek hmotnosti po přeměně jader musel přeměnit na energii

\begin{equation*}
    E = \Delta m c^2 = (m_{leva}-m_{prava})c^2
\end{equation*}

In [4]:
c = 299792458

E = (m_levo-m_pravo)*m_u*c**2
print(f"Energie získaná touto reakcí E = {E*1e12:,.2f} pJ = {E/1.602e-19*1e-6:,.2f} MeV")

Energie získaná touto reakcí E = 27.75 pJ = 173.25 MeV


Z jedné přeměny izotopu Uranu $^{235}\text{U}$ získáme 173.25 MeV energie. Kolik získáme z jednoho kilogramu Uranu $^{235}\text{U}$? Jenoduše vidělíme jeden koligram hmotností jednoho izotomu uranu a tím získáme počet atomů v jednom kilogramu. Tímto číslem poté vynásobíme energii získanou z jedné přeměny a získáme energii získanou štěpením z jednoho kilogramu uranu. 

In [5]:
N_U = 1/(m_U_235*m_u)
E_1_kg = N_U * E

In [6]:
E_sk = 1083708e15 # Roční spotřeb elektrické energie na slovensku v joulech

print(f"Na pokrytí roční spotřeby energie na Slovensku by bylo ptřeba (za velmi idealizovaných podmínek) {E_sk/E_1_kg*1e-3:,.2f} tun U235")

Na pokrytí roční spotřeby energie na Slovensku by bylo ptřeba (za velmi idealizovaných podmínek) 15,234.56 tun U235


15000 tun Uranu se může zdát hodně, ale při hustotě cca 20 tun/m $^3$ se jedná přibližne o objem 750 m $^3$ což je čtvrtina objemu olympijského bazénu.